docs/dev/listenbrainz-dumps.rst

```diff
@@ -67,3 +78,13 @@ Here is some example code to explain the mentioned way of parsing the listens du
 
 .. include:: ./dump_examples/read_listens_dump.py
    :code: python
+
+
+Incremental dumps
+=================
+
+ListenBrainz provides incremental data dumps that you can use to keep up to date with
```

Comment:
I think you should describe how we are ensuring continuity of the data dumps here. In particular, Full + partial ... partial = next full.